packages/netlify-cms-core/src/formats/toml.ts

```diff
@@ -1,17 +1,19 @@
 import toml from '@iarna/toml';
 import tomlify from 'tomlify-j0.4';
 import moment from 'moment';
-import AssetProxy from 'ValueObjects/AssetProxy';
+import AssetProxy from '../valueObjects/AssetProxy';
 import { sortKeys } from './helpers';
 
-function outputReplacer(key, value) {
+function outputReplacer(_key: string, value: unknown) {
   if (moment.isMoment(value)) {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore
     return value.format(value._f);
```

Comment:
What is this magic? Can it be done some other way?

Reply:
This was added in https://github.com/netlify/netlify-cms/pull/740/files#diff-319981fcd6e430ec283b2afe1a693e901030ccff607ec752487bae64f4e65fc2R8 

The only scenario I can think of that this code is triggered is a custom widget that reports a moment instance as a value. Our core widgets report either a JS Date instance or a string.

I couldn't find another way to get the `moment` instance format, so I think we can keep it this way.

Reply:
Well, we can't account for all of the Date libraries out there. Seem like it's a responsibility of the widget creator to convert value to js date.